static/app/components/lastCommit.spec.tsx

```diff
@@ -0,0 +1,30 @@
+import {render, screen} from 'sentry-test/reactTestingLibrary';
+
+import LastCommit from 'sentry/components/lastCommit';
+
+describe('LastCommit', function () {
+  const mockedCommit = TestStubs.Commit();
+  const mockedCommitTitle = '(improve) Add Links to Spike-Protection Email (#2408)';
+
+  it('renders', function () {
+    const wrapper = render(<LastCommit commit={mockedCommit} />);
+    expect(wrapper.container).toSnapshot();
+  });
+
+  it('links to the commit in GitHub', function () {
+    mockedCommit.repository.provider = {id: 'github'};
+    const mockedCommitURL = `${mockedCommit.repository?.url}/commit/${mockedCommit.id}`;
+
+    render(<LastCommit commit={mockedCommit} />);
+    expect(screen.getByText(mockedCommitTitle)).toBeInTheDocument();
+    expect(screen.getByText(mockedCommitTitle)).toHaveAttribute('href', mockedCommitURL);
+  });
+
+  it('displays the commit with its shortened ID if it has no message', function () {
+    mockedCommit.message = null;
```

Comment:
Doing this here (or rather, defining the mocked commit before any of the tests run) means that if the tests run in a different order, the 'links to the commit in GH' one will fail. Better to define the mocked commit separately in each test.

Reply:
ooh good point, thank you!